docs/DevOps.md

```diff
@@ -73,7 +73,7 @@ Any warnings or errors will be printed out to the console.
 
 Here are the steps to create a new release.
 
-1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java).
+1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/location/MainApp.java).
```

Comment:
Is all the `location` a placeholder for now?

Reply:
Nope. I think the refactoring screwed up all the links within the docs. Good catch!